vertx-core/src/main/java/io/vertx/core/http/impl/Http1xClientConnection.java

```diff
@@ -216,7 +216,7 @@ private HttpRequest createRequest(
 
   static CharSequence determineCompressionAcceptEncoding() {
     if (isBrotliAvailable() && isZstdAvailable()) {
```

Comment:
don't we need to add `isSnappyAvailable()` too ?

Reply:
Compared to `brotli` and `zstd`, [snappy is provided directly by Netty itself](https://github.com/netty/netty/blob/c176ce579106d5b8f5221cbdd8eb4a4ea70ebe1f/codec/src/main/java/io/netty/handler/codec/compression/Snappy.java), so we don't need to have a check that a library is on a class path.

Reply:
oh that's good to kow